UnityProject/Assets/Scripts/Managers/CargoManager.cs

```diff
@@ -520,6 +520,17 @@ public void ConfirmCart()
 			OnCartUpdate.Invoke();
 		}
 
+		public void SpendCredits(int amount)
+		{
+			if (!CustomNetworkManager.Instance._isServer)
```

Comment:
Let's continue following Unitystation's code styles and avoid using `!` unless we're inverting.

Reply:
yeah, follow unitystation code style.

Reply:
also you can just `CustomNetworkManager.IsServer`